Libplanet.Explorer/GraphTypes/ActionType.cs

```diff
@@ -13,13 +17,41 @@ public ActionType()
                 "Arguments",
                 resolve: ctx =>
                 {
+                    object ConvertValue(IValue value)
+                    {
+                        switch (value)
+                        {
+                            case Text text:
+                            case Integer integer:
+                            case Bencodex.Types.Boolean boolean:
+                            case Null n:
+                                return value;
+                            case Binary binary:
+                                return binary.ToString();
```

Comment:
[GraphQL.NET seems to have built-in converter for `byte[]`.](https://github.com/graphql-dotnet/graphql-dotnet/pull/1091)